### PR TITLE
CentOS support added

### PR DESCRIPTION
**Pull request by akilaMD**
- Added CentOS7 docker image
- Refactored dockerNodes.groovy file
- Passed CI on rocFFT
- Added to README file
- Lots of garbage commits that should be squashed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,51 +12,51 @@ In the project's Jenkinsfile, do the following to enable CentOS support:
 
 1) Define nodes by changing the label from the GPU architecture to include CentOS 7.
 
-Ex. Ubuntu: def nodes = new dockerNodes(['gfx900'], myproj)
```
Comment (saadrahim) [inline README.md]: Ubuntu 16 is the default OS

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 saadrahim
+Copyright (c) 2019 saadrahim
```
Comment (saadrahim) [inline LICENSE]: Change to AMD

Comment (saadrahim) [reply LICENSE]: In general, can you put in copyright AMD on all files.

File: src/com/amd/docker/dockerNodes.groovy [hunk 3]
```diff
@@ -18,11 +18,66 @@ class dockerNodes implements Serializable
...
+            }
+            else if(it.contains('centos7'))
+            {
+                dockerArray[it] = new rocDocker(
+                                baseImage: 'rocm/dev-centos-7:2.4',
+                                buildDockerfile: 'dockerfile-build-centos',
+                                installDockerfile: 'dockerfile-install-centos',
+                                runArgs: '--device=/dev/kfd --device=/dev/dri --group-add=video',
+                                buildArgs: '--pull',
+                                insideClosure: """
```
Comment (saadrahim) [inline src/com/amd/docker/dockerNodes.groovy]: insideClosure should be renamed to something that means something. infoCalls?

File: LICENSE (hunk 2 above)
Comment (akilaMD) [reply LICENSE]: Done. My next commit will have those changes.

**Pull request merged**